data/tutorials/guides/0tt_05_s_expressions.md

```diff
@@ -0,0 +1,267 @@
+
+The [`sexplib`](https://github.com/janestreet/sexplib) created by Jane Street provides support for S-expressions in OCaml. It was developped to provide support for S-expression to [Base](https://github.com/janestreet/base) and [Core](https://github.com/janestreet/core). However, S-expressions serialisation does not depend on these packages. In this tutorial, we only use `sexplib`. You can install it using opam.
+```shell
+$ opam install sexplib
+```
+
+## S-Expression Format
+
+S-expressions are represented as **nested lists** surrounded by parenthesis. Each element of the list can be either an **atom** or another nested list. The format of an S-expression follows a specific structure, which is crucial to understand when manipulating S-expressions.
+
+Here are a few examples:
+
+```lisp
+()
+(a)
+(a b)
+(a b c)
+(a (b 0) (c 1.2))
+```
+
+Each of the individual symbols are atoms. An **atom** represents a basic data value. Atoms can be of different types, including integers, floating-point numbers, strings, or Booleans. In OCaml, atoms are represented as their corresponding OCaml data types. For example:
+
+  + Integer: `42`
+  + Float: `3.14`
+  + String: `"Hello, world!"`
+  + Boolean: `true`
+
+
+Inside each set of parentheses is a nested list of S-Expressions. A nested list is used to represent the **hierarchical structure of S-Expressions**. Each element can be an atom or another nested list. The elements within the list are separated by spaces.
+
+In OCaml, S-Expressions are defined as follows:
+
+```ocaml
+type sexp =
+  | Atom of string
+  | List of sexp list (** Nested list. *)
+```
+
+This type and the basic operations associated to this type are defined in the [sexplib](https://github.com/janestreet/sexplib) package.
+
+Note: the syntax of [Dune configuration files](https://dune.readthedocs.io/en/latest/reference/lexical-conventions.html) is similar to S-expressions.
+
+## Serialising
+
+> Serialisation is the process of translating a data structure or object state into a format that can be stored (e.g., files or data buffers) or transmitted and reconstructed later.
+
+Here it means we can translate OCaml data types into S-expressions and then into a more machine readable format.
+
+### Constructing
+
+The [Core](https://github.com/janestreet/core) package provides many ways to deal with S-expressions, a `Sexp` module to deal with the S-expressions themselves, and `sexp_of_t` / `t_of_sexp` functions provided by data type modules (like `String`, `List`, etc.) that allow users to convert OCaml data to and from S-expressions.
+
+The provided examples are intended to be excuted in UTop. Here is the initial setup:
+
+```ocaml
+# require "sexplib";;
+# open Sexplib;;
```

Comment:
I can't get this to work either, likely for the same reason as whatever's happening above.

This is capitalised but li 69 `sexplib` isn't. Is this intentional?